lms/djangoapps/course_api/blocks/transformers/blocks_api.py

```diff
@@ -65,3 +67,5 @@ def transform(self, usage_info, block_structure):
         BlockCountsTransformer(self.block_types_to_count).transform(usage_info, block_structure)
```

Comment:
You will need to update the Write and Read versions of this Transformer, in a 2-phase deployment.  Please see https://openedx.atlassian.net/wiki/spaces/MS/pages/154861855/Block+Structure+Cache+Invalidation+Proposal#BlockStructureCacheInvalidationProposal-VersioningScheme:read-versionversuswrite-version

Reply:
The 2-phase deployment is not necessary as this transformer is not collecting any data. Additionally, it is not registered as a new transformer because it is contained in BlocksAPITransformer umbrella transformer and is mutating the data of StudentViewTransformer.